ansible-collection-redhatci-ocp.spec

```diff
@@ -3,7 +3,7 @@
 %global forgeurl https://github.com/%{org}/%{repo}
 
 Name:           %{repo}
-Version:        0.7.EPOCH
+Version:        0.8.EPOCH
```

Comment:
LGTM, just for my understanding, what is the reason to update the version here? I'm curious in case I've been missing something in my previous changes to this repo. Thanks.